hilti/runtime/include/types/bitfield.h

```diff
@@ -23,15 +23,26 @@ struct isBitfield {};
 /// printing.
 template<typename... Ts>
 struct Bitfield : public trait::isBitfield {
-    std::tuple<std::optional<Ts>...> value;
+    using value_type = std::tuple<std::optional<Ts>...>;
+
+    Bitfield(value_type v = {}) : value(std::move(v)) {}
```

Comment:
Its a bit unfortunate that we add a non-`explicit` ctr which will make it easier to subvert our type system, but it seems just replacing it with a direct ctr from bitfield field values does not work (another motivation for that: get rid of `make_bitfield` since we now have a proper class here):

```cpp
Bitfield(Ts... xs) : value(std::make_tuple(xs...)) {}
```

Reply:
We indeed need this non-explicit, that's part of the original issue here.